test/apps/iodemo/io_demo.cc

```diff
@@ -1309,8 +1310,13 @@ class DemoServer : public P2pDemoCommon {
         w->init(this, conn, msg->sn, msg->conn_id, iov,
                 &conn_stat.completions<IO_WRITE>());
         conn_stat.bytes<IO_WRITE>() += msg->data_size;
-        conn->recv_am_data((*iov)[0].buffer(), (*iov)[0].size(),
-                           (*iov)[0].memh(), data_desc, w);
+        if (!ucx_am_is_rndv(data_desc)) {
+            conn->recv_am_data(NULL, 0,
```

Comment:
wrap lines

Reply:
do you mean combining line 1314 & 1315 into one line?

Reply:
yes